.github/workflows/ci.yml

```diff
@@ -153,5 +169,18 @@ jobs:
         run: |
           curl -f -s -X POST -H "Content-Type: application/json" -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $BACKPACK_DOCS_DEPLOY_TOKEN" -d '{"ref":"master","inputs":{"repo":"backpack-ios"}}' https://api.github.com/repos/Skyscanner/backpack-docs/actions/workflows/ci.yml/dispatches
 
+  comment_on_pr:
+    needs: CI
+    name: Comment on PR
+    if: github.ref != 'refs/heads/master'
+    runs-on: ubuntu-20.04
 
-
+    steps:
+      - name: Link to the pull request build
+        uses: unsplash/comment-on-pr@master
+        if: github.ref != 'refs/heads/master'
+        with:
+          msg: 'Visit https://backpack.github.io/ios-prs/${{ env.PR_NUMBER }} to see the docs for this PR.'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
```

Comment:
`unsplash/comment-on-pr@master` only works on linux, so what this does is waits for CI to finish (`needs: CI`), and then spins up a linux job to comment on the PR

Reply:
smort!